src/plugins/curator/components/gallery-media/utils/moveNodeByGallerySequence.test.js

```diff
@@ -0,0 +1,155 @@
+import test from 'tape';
+import isEqual from 'lodash/isEqual';
+import AssetV1Mixin from '@triniti/schemas/triniti/dam/mixin/asset/AssetV1Mixin';
+import AssetId from '@triniti/schemas/triniti/dam/AssetId';
+
+import moveNodeByGallerySequence from './moveNodeByGallerySequence';
+
+const assetSchemas = AssetV1Mixin.findAll();
+const imageAssetSchema = assetSchemas[4];
```

Comment:
@hunyoboy tests should just pull in the acme schema you want direct.  references array index from findAll is fragile.  if that's not doable then use the image asset mixin to do the resolution, then you can use findOne